molecule/default/verify.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Check whether packages are installed
+  hosts: all
+  become: yes
+
+  tasks:
+    - name: Gather package facts
+      package_facts:
+        manager: auto
+
+    - name: Verify Packages
+      assert:
+        that: "'{{ item }}' in ansible_facts.packages"
+      with_items:
+        - git
+        - curl
+        - mcrypt
+        - htop
+        - iotop
+        - make
+        - multitail
+        - ncdu
+        - nload
+        - nmon
+        - pv
+        - tree
+        - unzip
+        - bc
+        - screen
+      when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu'
+
+    - name: Verify Packages
+      assert:
+        that: "'{{ item }}' in ansible_facts.packages"
+      with_items:
+        - git
+        - curl
+        - libselinux-python3
```

Comment:
The defaults file is still specifying the python2 version

https://github.com/AcroMedia/ansible-role-essentials/blob/add-automated-tests/defaults/main.yml#L5